src/__tests__/TuxAnimations-test.js

```diff
@@ -35,42 +40,47 @@ describe('Animations', function () {
 
   it("applies a class of 'fly' for a fly transition", function () {
     ideaFlyComponent = TestUtils.renderIntoDocument(
-      <Fly key={mocks.ideas[0].id}>
-        <Idea key={mocks.ideas[0].id} />
-      </Fly>
+      <Fly key={mocks.ideas[0].id} />
     );
+    ideaFlyComponent.setProps({children: <Idea key={mocks.ideas[0].id} /> })
```

Comment:
Explain why we need to use `setProps` here and why we are setting the `children` prop.